src/components/common/ratings.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import Rating from '@material-ui/lab/Rating';
+import { makeStyles } from '@material-ui/core/styles';
```

Comment:
I'm not sure that we are allowed to use Material UI here. I can't remember off the top of my head.

Reply:
Yes, since we haven't been able to implement rating groomers yet, we should probably remove the stars altogether, and I think that is the only place you are using material-ui.